2001-3000/LIT2892RepCh173.xml

```diff
@@ -58,14 +54,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <bibl>
                   <ptr target="bm:Chaine1913Rep"/>
                   <citedRange unit="item">173</citedRange>
+                  <note>
+                     <persName ref="PRS3082Chaine"/> was mistakenly identified this a malkǝʾ-hymn for
+                     <persName ref="PRS9848Walatta"/> in 
+                     <bibl>
+                        <ptr target="bm:Chaine1913RepPart1"/>
+                        <citedRange unit="page">202</citedRange>
+                     </bibl>, but correctly indentified this as a malkǝʾ-hymn to 
+                     <persName ref="PRS3698ehetaKr"/> in 
+                     <bibl>
+                        <ptr target="bm:Chaine1912Catalogue"/>
+                        <citedRange unit="page">57</citedRange>
+                        <citedRange unit="number">88</citedRange>
+                     </bibl>.
+                  </note>
                </bibl>
             </listBibl>
          </div>
          <div type="edition">
             <div type="textpart" subtype="incipit" xml:lang="gez">
                <ab>
-                  ሰላም፡ ለዝክረ፡ ስምኪ፡ ሞገስ፡ ሃይማኖት፡ ዘተረሰየ ።
-               </ab>
+                  <hi rend="rubrication">ሕይወተ፡ ወላዲ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ </hi>ከሣቴ፡ ኅቡኣት፡ በበጾታ፡ ለነቢያት፡ ቅድመ፡ ወለ<hi rend="rubrication">ሐዋርያት፡ ሕዝበ፡ ኤፍራታ፨ እምዝቀ፡ ሥ</hi>ምረትከ፡ እቅዳሕ፡ ለ<hi rend="rubrication">እኅተ፡ ክርስቶስ፡ ወይነ፡ ማ</hi>ኅሌታ፨ይሰወጥ፡ ለልሳንየ፡ ቅድስናከ፡ ውስቴታ፤ ወይጸልለኒ፡ እምሰምይ፡ አክናፊከ፡ መንታ፨
+                  <note>This incipit was taken directly from <ref type="mss" corresp="BNFabb88"/>, differing from what is given in <bibl><ptr target="bm:Chaine1912Catalogue"/><citedRange unit="page">57</citedRange><citedRange unit="number">88</citedRange></bibl></note>.
+              </ab>
             </div>
```

Comment:
as discussed https://github.com/BetaMasaheft/Documentation/issues/2640
you can also add the relation 

`<relation name="skos:exactMatch" active="LIT2892RepCh173" passive="https://w3id.org/malke/2892/"></relation> 
`
and add the database in the bibliography.

It is up to you - and @smaugustine -  if it is also possible to insert the `div type="edition" `from https://malkeagubae.com/api/xml/works/2892.xml (providing the source of data of course)

Reply:
I'm happy to see any of the data used!